packages/api-client/src/views/Request/libs/auth.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  type SecurityScheme,
+  securitySchemeExampleValueSchema,
+} from '@scalar/oas-utils/entities/spec'
+
```

Comment:
Moved out some functions into another file and it still wasn't enough to hit the line limit 😓